libs/utils/utils.js

```diff
@@ -712,6 +712,17 @@ export function decorateLinks(el) {
     if (a.href.includes(copyEvent)) {
       decorateCopyLink(a, copyEvent);
     }
+    // Append aria-label
+    const pipeRegex = /\s?\|\s?/;
+    if (pipeRegex.test(a.textContent) && !/\.[a-z]+/i.test(a.textContent)) {
```

Comment:
How would this work for links with `.mp4`? Do we need to handle such an use-case?

Reply:
There's an example of that on the [test page](https://links-aria-label--milo--adobecom.hlx.page/drafts/ramuntea/accessibility/aria-label-with-icon?martech=off). If the link has an extension, the aria label logic will ignore it. The alternative would be to modify the auto-block logic to ignore anything that qualifies as an auto-block from having the aria-label logic applied as well